Parser.fs

```diff
@@ -585,6 +585,22 @@ let parseConstraint : Parser<ViewSignature * Expression option, unit> =
             (fun d _ v -> (d, v))
     .>> pstring ";"
 
+
+/// parse an exclusivity constraint
+let parseExclusive : Parser<List<strFunc>, unit> = 
+    pstring "exclusive" >>. ws
+    // ^- exclusive ..  
+    >>. parseDefs (parseFunc parseIdentifier)
```

Comment:
Does this allow whitespace between the last func and the `;`?  There's a parser (`wsSemi` I think?) that ensures this